glusterd2/commands/volumes/volume-create-txn.go

```diff
@@ -92,6 +92,15 @@ func populateSubvols(volinfo *volume.Volinfo, req *api.VolCreateReq) error {
 			if subvolreq.ReplicaCount != 2 || subvolreq.ArbiterCount != 1 {
 				return errors.New("for arbiter configuration, replica count must be 2 and arbiter count must be 1. The 3rd brick of the replica will be the arbiter")
 			}
+			var abriterBrickCount uint16
+			for _, b := range subvolreq.Bricks {
+				if b.Type == "arbiter" {
```

Comment:
`strings.Lower` is required here

Reply:
@Madhu-1 Can we add this checks and other if conditions reagrding replicaCount, disperseCount, etc, inside 
switch subvolreq.Type{} in the previous lines.

Reply:
@vpandey-RH i think it's not required, @aravindavk  any comments?